client/state/selectors/is-jetpack-onboarding-step-completed.js

```diff
@@ -0,0 +1,48 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+import { translate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import createSelector from 'lib/create-selector';
+import { getJetpackOnboardingSettings } from 'state/selectors';
+import { JETPACK_ONBOARDING_STEPS as STEPS } from 'jetpack-onboarding/constants';
+
+export const isJetpackOnboardingStepCompleted = createSelector(
+	( state, siteId, stepName ) => {
+		const settings = getJetpackOnboardingSettings( state, siteId );
+
+		if ( ! settings ) {
+			return null;
+		}
+
+		switch ( stepName ) {
+			case STEPS.SITE_TITLE:
+				const titleModified = get( settings, 'siteTitle', '' ) !== '';
```

Comment:
Is the default title really empty, even tho the default desc isn't?

Reply:
Yes - I verified by doing a fresh WordPress install - title field is empty and there is no validation to ensure it isn't upon site installation. There also isn't a default value that overrides this, so whenever you finish installing your WP, the site title ends up as an empty string (`""`). It's not `false` though (the option is there, just has an empty string value), so we should be good by checking the empty string.